src/sentry/static/sentry/app/views/releases/pullRequestLink.jsx

```diff
@@ -51,10 +38,10 @@ class PullRequestLink extends React.Component {
       );
     }
 
-    return url ? (
+    return pullRequest.externalUrl ? (
       <a
```

Comment:
aside: use `app/components/ExternalLink` - it includes `rel="noreferrer noopener"` for security purposes.